src/v2/Apps/Conversation/Components/Conversations.tsx

```diff
@@ -17,22 +21,48 @@ const Container = styled(Box)`
   `};
 `
 
+const SpinnerContainer = styled.div`
+  width: 100%;
+  height: 100px;
+  position: relative;
+`
+
+export const PAGE_SIZE: number = 25
```

Comment:
**suggestion:** Conversation previews take up a good bit of space in the view with ~7-10 being able to fit on a desktop screen at a time. 15 should be plenty for most cases and especially help cut down on the data being pulled for mobile. 

```suggestion
export const PAGE_SIZE: number = 15
```